routes/posts.js

```diff
@@ -55,6 +54,20 @@ router.get('/:id', authorization, async (req, res) => {
   }
 })
 
+// delete selected post
+router.delete('/delete/:post_id', authorization, async (req, res) => {
+  try {
+    const deletePost = await db.query('DELETE FROM posts WHERE id=$1', [
+      req.params.post_id,
+    ])
+
+    return res.status(200).json(`Post ${req.params.id} Deleted`)
+  } catch (err) {
+    console.error(err.message)
+    return res.status(500).json('Server Error')
```

Comment:
from now on, i'd like to see more descriptive error messages and status', it'll help a lot with debugging in the future instead of just returning a 500 for everything with a generic server error message. As your code base gets bigger and more people work on your code, it'll be hard to track down such errors